zerobuf/Zerobuf.h

```diff
@@ -10,6 +10,7 @@
 #include <zerobuf/api.h>
 #include <zerobuf/Types.h>
 #include <servus/uint128_t.h>
+#include <zerobuf/jsoncpp/json/json.h>
```

Comment:
I would keep it internally then, as it is also used internally only now. The implementation of the string version should reuse the Json::Value one (it's C&P now). For symmetry, add fromJSON( Json::Value )?